pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+
+import pytest
+
+from pulp_smash.config import get_config
+
+from pulpcore.client.pulpcore import ApiClient, TasksApi
+
+
+@pytest.fixture
+def pulpcore_client():
+    cfg = get_config()
+    configuration = cfg.get_bindings_config()
+    return ApiClient(configuration)
+
+
+@pytest.fixture
+def tasks_api_client(pulpcore_client):
+    return TasksApi(pulpcore_client)
+
+
+@pytest.fixture
+def async_monitor_task(tasks_api_client):
```

Comment:
I still fail to see the value in this being a "fixture".

Reply:
I agree. I'm not finished with this one yet. The issue I'm struggling with is that if this becomes a normal function then I can't access the chain of fixtures it depends on tasks_api_client -> pulpcore client fixtures. I'm going to look at this more today.

Reply:
Oh right. There is a point.